src/main.js

```diff
@@ -1,16 +1,21 @@
-// query selector variables go here 👇
-//poster attributes
+//~~~~~~~~~~~~~~~~~~~~~~~~QUERY SELECTOR VARIABLES 👇~~~~~~~~~~~~~~~~~~~~~~~~
+
+  //poster attributes
 var title = document.querySelector('.poster-title')
 var quote = document.querySelector('.poster-quote')
 var image = document.querySelector('img')
 
-//pages
+//should the rest of these variables (pages and buttons) be global, or
```

Comment:
My understanding from the rubric was that we should have no global variables other than query selectors, so I think these are all ok to leave in this section.